docs/development-environment.md

```diff
@@ -38,24 +38,25 @@ $ pip install -r requirements/develop.pip
 $ pip install -e .
 ```
 
-### Macos Big Sur caveat
+### Mac OS caveat
 
 If installing `cryptography` fails:
 
 ```
 brew install openssl
```

Comment:
We should specify openssl@1.1 here too.

For reference, udata is using [cryptography 2.8](https://cryptography.io/en/2.8/installation/) not tested with openssl@3.

```suggestion
brew install openssl@1.1
```

Reply:
Yep, done in latest commit ;-)

Reply:
Is it pushed already ? I'm not seeing it for this line.

Reply:
Yeah, right, sorry.

Reply:
Thanks ! 🙏🏻